JavaScript/JavaScriptSDK/Telemetry/PageViewPerformance.ts

```diff
@@ -137,12 +146,22 @@ module Microsoft.ApplicationInsights.Telemetry {
         }
 
         public static getDuration(start: any, end: any): number {
-            var duration = 0;
+            var duration = undefined;
             if (!(isNaN(start) || isNaN(end))) {
                 duration = Math.max(end - start, 0);
             }
 
             return duration;
         }
+
+        public static maxDurationExceeded(...durations: number[]): boolean {
+            for (var i = 0; i < durations.length; i++) {
```

Comment:
why is it public? Doesn't seem like it is called anywhere outside PageViewPerformance class.